proto/config/edge_node_cluster.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright(c) 2024 Zededa, Inc.
+// All rights reserved.
+
+syntax = "proto3";
+
+package org.lfedge.eve.config;
+option go_package = "github.com/lf-edge/eve-api/go/config";
+option java_package = "org.lfedge.eve.config";
+
+import "evecommon/acipherinfo.proto";
+
+message EdgeNodeCluster {
+  // cluster name, in case it has multiple cluster on the same site
+  string cluster_name = 1;
+
+  // cluster UUID
+  string cluster_id = 2;
+
+  // Cluster-Interface, for example "eth1"
+  string cluster_interface = 3;
+
+  // the 'cluster-prefix' IP address of the 'Cluster-Interface', 10.244.244.2/28
+  string cluster_ip_prefix = 4;
+
+  // This device is an 'Agent' node
+  bool is_agent = 5;
```

Comment:
Should this be `is_agent` or `is_worker`? I think maybe `worker`? The `agent`/`server` language is a k3s one; `control plane`/`worker` is generic Kubernetes. We _are_ using k3s here, but that is an implementation detail. I think the API should be generic to Kubernetes.

Reply:
Sure. we can change it to is_worker_node. although we have things like join_server_ip, we don't want to be too kubernetes as join_control_plan_node_ip, and 'master' should not be used these days :-)

Reply:
Yeah. I think server is fine though. Lots of people use "server" as thing to talk to, and even refer to "control plane server" or "master server". Besides, the only thing I can join is a control plane server anyways.

Reply:
This still needs to be addressed

Reply:
Thanks! I missed it yesterday.  Done, now is `is_worker_node`.